lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -60,12 +60,12 @@
 LOGGER = getLogger()
 
 TIMEOUT_CODE = 124
-TIMEOUT_WARNING = '\n'.join([
-    "mympirun has been running for %s seconds without seeing any output.",
-    "This may mean that your program is hanging, please check and make sure that is not the case!",
-    '',
-    "If this warning is printed too soon and the program is doing useful work without producing any output,",
-    "you can increase the timeout threshold via --output-check-timeout (current setting: %s seconds)"])
+TIMEOUT_WARNING = """mympirun has been running for %s seconds without seeing any output.
+                  This may mean that your program is hanging, please check and make sure that is not the case!
```

Comment:
@Caylo if you indent like this, the indentation is also included in the printed warning?